packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -200,13 +245,15 @@ export class TokenDetectionController extends StaticIntervalPollingController<To
    * @param options.interval - Polling interval used to fetch new token rates
    * @param options.getBalancesInSingleCall - Gets the balances of a list of tokens for the given address.
    * @param options.trackMetaMetricsEvent - Sets options for MetaMetrics event tracking.
+   * @param options.useAccountsAPI - Feature Switch for using the accounts API when detecting tokens (default: true)
    */
   constructor({
     interval = DEFAULT_INTERVAL,
     disabled = true,
     getBalancesInSingleCall,
     trackMetaMetricsEvent,
     messenger,
+    useAccountsAPI = true,
```

Comment:
Big proponent of feature switching (in case this PR gets too large, or we need to turn this feature off).

Happy to remove otherwise.

Reply:
This is enabled by default